server.js

```diff
@@ -48,6 +48,7 @@ const didyouknowResponses = [
   'did you know chris has only one chrome extention and its his',
   'did you know chris is getting annoyed rn with all the did you know trivia',
   'did you know chris once streamed for 15 hours'
+  'did you know chirs would walk out of an interview if you ask him about shift and unshift array methods in js'
```

Comment:
Typo: 'chirs'

Reply:
Instead of "...ask him about shift and unshift..." can we do "...ask him about the difference between call and apply..."